eng/submit-source-build-release-pr.sh

```diff
@@ -33,7 +33,7 @@ OPTS=$(getopt --options $SHORT --long $LONG --name "$0" -- "$@")
 if [ $? != 0 ] ; then echo "Failed to parse options." >&2 ; exit 1 ; fi
 eval set -- "$OPTS"
 
-global_json_path='src/SourceBuild/tarball/content/global.json'
+global_json_path='src/SourceBuild/content/global.json'
```

Comment:
We use this pipeline for 6.0 and 7.0 release processes, which still has this path.

Maybe we need to fork `dotnet/source-build` so that we can have updated release pipelines for the VMR process. @MichaelSimons thoughts?

Reply:
Do you always use this script from main or can you maybe run it from the 7.0 branch of source-build?

Reply:
I personally like the idea of maintaining a single copy of the engineering infra we utilize for releases if it reduces the maintenance burden.  If conditioning the logic per branch is not feasible then we should branch.  I think it would be confusing however if we use main for .NET 8.0 and then release/7.0 for both 7.0 and 6.0.

Reply:
@premun could you condition this later in the script on the value of `sdkVersion`?